protos/xyz/block/ftl/v1/ftl.proto

```diff
@@ -51,12 +51,20 @@ service VerbService {
   rpc Call(CallRequest) returns (CallResponse);
 }
 
+enum DeploymentChangeType {
+  DEPLOYMENT_ADDED = 0;
+  DEPLOYMENT_REMOVED = 1;
+  DEPLOYMENT_CHANGED = 2;
+}
```

Comment:
@alecthomas add this to support notifying when items are removed from the schema. I'm not sure if this was working, but curious to get your thoughts on this approach.